src/styles/blocks/card.scss

```
@@ -0,0 +1,91 @@
+.card {
+  max-width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+
+  &__img {
+    max-width: 160px;
+    height: 134px;
+    display: flex;
+    margin: 0 auto;
+  }
+
+  &__heading {
+    margin: 40px 0 0;
+    font-style: normal;
+    line-height: 18px;
+    font-weight: 500;
+    font-size: 12px;
+    color: #060b35;
+  }
+
+  &__code {
+    font-style: normal;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: #616070;
+    margin: 4px 0 0;
+  }
+
+  &__rating {
+    margin: 16px 0 0;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    &-count {
+      font-weight: 400;
+      font-size: 10px;
+      line-height: 14px;
+      color: #060b35;
+    }
+  }
+
+  &__price {
+    margin: 24px 0 0;
+    display: flex;
+    justify-content: space-between;
+
+    &-text {
+      font-weight: 400;
+      font-size: 12px;
+      line-height: 18px;
+      color: #616070;
+    }
+
+    &-amount {
+      font-weight: 700;
+      font-size: 16px;
+      line-height: 18px;
+      color: #060b35;
+    }
+  }
+
+  &__btn {
+    margin: 16px 0 0;
+    display: flex;
+    padding: 11.5px 55.5px 11.5px 54.5px;
+    background-color: #00acdc;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+    text-align: center;
+    text-transform: uppercase;
+    border-radius: 5px;
+    color: #fff;
+    border: 1px solid #fff;
```

```suggestion
    border: 1px solid transparent;
```
there could be your issue with a link botton